libs/SalesforceSDK/src/com/salesforce/androidsdk/accounts/UserAccount.java

```diff
@@ -752,9 +831,10 @@ public int hashCode() {
 	/**
 	 * Returns a JSON representation of this instance.
 	 *
+	 * @param additionalOauthKeys: values to pull from the additionalOauthValues
 	 * @return JSONObject instance.
 	 */
-	public JSONObject toJson() {
+	JSONObject toJson(List<String> additionalOauthKeys) {
```

Comment:
The public toJson takes no argument and calls this one with the additionalOauthKeys returned by SalesforceSDKManager. Having this method allows for simpler tests.